tests-web/www/js/files.js

```diff
@@ -52,3 +52,20 @@ const forEachFile = (dt, f) => {
     }
   }
 };
+
+export const loadFile = (name) =>
+  fetch(name)
+    .then((body) => body.text())
+    .catch((err) => console.warn(err));
+
+export function downloadGeneratedFile(filename, textInput) {
```

Comment:
I lifted this from some stackoverflow page. FWIW, I've seen this not work when passing bigger (50MB?) URL-encoded binary files from Rust. But seems fine for our small xml files for now.